holidays/locale/en_US/LC_MESSAGES/DK.po

```diff
@@ -5,60 +5,67 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Python Holidays 0.20\n"
 "POT-Creation-Date: 2023-02-15 08:13-0800\n"
-"PO-Revision-Date: 2022-12-16 12:25-0800\n"
-"Last-Translator: Arkadii Yakovets <ark@cho.red>\n"
+"PO-Revision-Date: 2023-04-09 13:48+0300\n"
+"Last-Translator: ~Jhellico <jhellico@gmail.com>\n"
 "Language-Team: Python Holidays localization team\n"
 "Language: en_US\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "Generated-By: pygettext.py 1.5\n"
 "X-Generator: Poedit 3.2.2\n"
 
-#: holidays/countries/denmark.py:36
+#. New Year's Day.
+#: holidays/countries/denmark.py:40
 msgid "Nytårsdag"
 msgstr "New Year's Day"
 
-#: holidays/countries/denmark.py:37
-msgid "Palmesøndag"
-msgstr "Palm Sunday"
-
-#: holidays/countries/denmark.py:38
+#. Holy Thursday.
+#: holidays/countries/denmark.py:43
 msgid "Skærtorsdag"
 msgstr "Maundy Thursday"
 
-#: holidays/countries/denmark.py:39
+#. Good Friday.
+#: holidays/countries/denmark.py:46
 msgid "Langfredag"
 msgstr "Good Friday"
 
-#: holidays/countries/denmark.py:40
+#. Easter Sunday.
+#: holidays/countries/denmark.py:49
 msgid "Påskedag"
 msgstr "Easter Sunday"
 
-#: holidays/countries/denmark.py:41
+#. Easter Monday.
+#: holidays/countries/denmark.py:52
 msgid "Anden påskedag"
 msgstr "Easter Monday"
 
-#: holidays/countries/denmark.py:42
+#: holidays/countries/denmark.py:58
 msgid "Store bededag"
-msgstr "Great day of prayer"
+msgstr "General Prayer Day"
 
-#: holidays/countries/denmark.py:43
+#. Ascension Day.
+#: holidays/countries/denmark.py:62
 msgid "Kristi himmelfartsdag"
 msgstr "Ascension Day"
 
-#: holidays/countries/denmark.py:44
+#. Whit Sunday.
+#: holidays/countries/denmark.py:65
 msgid "Pinsedag"
 msgstr "Whit Sunday"
 
-#: holidays/countries/denmark.py:45
+#. Whit Monday.
+#: holidays/countries/denmark.py:68
 msgid "Anden pinsedag"
 msgstr "Whit Monday"
 
-#: holidays/countries/denmark.py:46
+#. Christmas Day.
+#: holidays/countries/denmark.py:71
 msgid "Juledag"
 msgstr "Christmas Day"
 
-#: holidays/countries/denmark.py:47
+#. Second Christmas Day.
+#: holidays/countries/denmark.py:74
 msgid "Anden juledag"
-msgstr "Boxing Day"
+msgstr "Second Christmas Day"
```

Comment:
```suggestion
msgstr "Second Day of Christmas"
```

or Christmas Second Day?

Reply:
I thought for a long time with this (Dec 26 holiday in various countries) what would be the most correct. So now I choose Christmas Second Day. 👍

Reply:
I like "Second Day of Christmas" better ;)

Reply:
I definitely trust your language skills and experience!